src/pages/guides/changelog/index.md

```diff
@@ -15,6 +15,21 @@ contributors:
 
 # Changelog
 
+## [4.16.11] - 2024-11-18
+
+### Enhancements
+
+- [`CCEverywhere.appConfig.imageStyleReference`](../../v4/shared/src/types/module/AppConfig.types/interfaces/TextToImageAppConfig.md#properties): in a `createImageFromText()` module, assets can be passed as style reference from host application via `appConfig`.
+- [`CCEverywhere.appConfig.imageCompositionReference`](../../v4/shared/src/types/module/AppConfig.types/interfaces/TextToImageAppConfig.md#properties): in a `createImageFromText()` module, assets can be passed as composition reference from host application via `appConfig`.
```

Comment:
Can we add a sample here?

Reply:
I'm not sure it would be a good idea: we've never used the changelog as a snippet repository (I checked the add-ons, too), and given the params involved in the Embed SDK in general, it could lead to lengthy snips.
I'll rather prefer the long-term plan of adding comments and examples in the source code, and let typedoc parse and display them in the API Ref.